### ISSUE TITLE
Completed module 2 assignments

### ISSUE THREAD

**Issue by alecumitu** [rbtl-fs24/md-02-assignments-alecumitu#1]
@larnsce 
What did the z distribution say to the t distribution?

**Comment by larnsce** [comment]
Thank you for working through the assignment, @alecumitu. Could you double-check that you did the commit to track your changes and then also push to GitHub?rb

It’s Steps 2 & 3 as described on the website and also in the assignments itself: https://rbtl-fs24.github.io/website/assignments/md-02/am-02-2-data-visualisation.html#step-2-commit-and-push-your-changes

**Comment by alecumitu** [comment]
Dear Lars,

Thank you for writing me.

I also wrote you about this last night from my ethz mail. I followed all
the steps regarding the commit and push and it tells me that everything is
up to date. However it still does not appear in the repository. I tried
opening the assignment twice in posit.cloud to commit and push again but
still nothing.

What should I do? I got this message after committing in case it gives you
any hint about what i did wrong.

Best,
Alexandru



Lars Schöbitz ***@***.***> schrieb am Mi. 6. März 2024 um
14:48:
